chesster.js

```diff
@@ -365,190 +171,34 @@ function prepareCommandsMessage(){
         "        mods summon (lonewolf)]    ! summon the mods\n" +
         "    [ faq ]                        ! a document of frequently asked questions\n" + 
         "    [ registration | sign up ]     ! registration form to play in our league\n" +
-        "    [ changelog ]                  ! list a log of changes to Chesster \n" +
         "    [ source ]                     ! github repo for Chesster \n" +
         "```\n";
 }
 
-function sayCommands(convo){
-    convo.say(prepareCommandsMessage());
-}
-
-controller.hears([
-    'commands', 
-	'command list'
-],[
-	'direct_mention', 
-	'direct_message'
-], function(bot,message) {
-    bot_exception_handler(bot, message, function(){
-        bot.reply(message, prepareCommandsMessage());
+chesster.hears({
+    patterns: [
+        'commands', 
+        'command list',
+        'help',
+    ],
+    messageTypes: [
+        'direct_mention', 
+        'direct_message'
+    ]
+},
+function(bot,message) {
+    bot.startPrivateConversation(message, function (response, convo) {
```

Comment:
I recall, regarding the handler's inability to deal with failures in asynch contexts.